sites/bizbash/server/styles/index.scss

```diff
@@ -64,3 +64,45 @@ $theme-carousel-caption-background: rgba(0, 0, 0, .75);
     content: "*";
   }
 }
+
+.top-list {
+  &__title {
+    @include theme-page-body-spacing();
+    @include theme-content-heading(h4);
+  }
+  &__row {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-bottom: map-get($spacers, block);
+  }
+  &__column {
+    margin-bottom: map-get($spacers, block);
+    @include make-col-ready();
+    @include media-breakpoint-up(md) {
+      @include make-col(4);
+    }
+  }
+  &__link {
+    display: block;
+  }
+  &__overlay {
+    small {
```

Comment:
Don't target to `<small>` - this should be a separate element with all styles applied